examples/specs/bubble_health_income.json

```diff
@@ -0,0 +1,23 @@
+{
+  "description": "A bubble plot showing the correlation between health and income for 187 countries in the world.",
```

Comment:
I'm actually not sure where the original data comes from. I downloaded the dataset from this url (https://raw.githubusercontent.com/OpenNewsLabs/onechart-twelvechartinglibraries/master/Vega-Lite/data.csv) and added it to `vega-datasets` repo.

I think we should definitely credit gapminder -- perhaps by citing this url (http://www.gapminder.org/tools/bubbles#_)?

But then if we are claiming that we are recreating their graph, I guess we should make our example look more like their graph? I can do the following:

* change x from "income" to "GDP/capita in $"
* change y from "health" to "life expectancy in years"
* use color to encode continent

I may need to make some changes in `vega-datasets` too.

Should I do the above things?